src/ui-components/cards/horizontal-label-value.component.tsx

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import styles from "./horizontal-label-value.css";
+import { useTranslation } from "react-i18next";
 
 export default function HorizontalLabelValue(props: HorizontalLabelValueProps) {
+  const { t } = useTranslation();
   return (
     <div className={styles.root}>
       <label
         className={props.labelClassName || "omrs-type-body-regular"}
         style={props.labelStyles}
       >
-        {props.label}
+        {t(props.label)}
```

Comment:
I think this is dangerous. The user of the component does not know that the input will be localized. I think that all props are expected as localized. Otherwise, we'll have a problem.

Example: If React would do what we do here now, we would only supply `<div>my text</div>`, which is then essentially equivalent to `<div><Trans>my text</Trans></div>`, however, what if we already supplied (by not knowing) a translated string? It would be `<div><Trans><Trans>my text</Trans></Trans></div>`. The only way out of this is to state "this component takes the input as-is - if you want it translated, then please translate".

Reply:
I agree. I've amended it so that it receives already translated strings in the latest commit.